src/MultipartStreamBuilder.php

```diff
@@ -93,7 +93,7 @@ public function addData($resource, array $headers = [])
      *
      * @param string                                                        $name     the formpost name
```

Comment:
```suggestion
     * @param string                                                    $name     the formpost name
```

Reply:
so annoying :D

Reply:
yeah its nice to read when aligned, but not nice for the diffs at all.